doc/source/user_guide/tutorials/mathematics/basic_maths.rst

```diff
@@ -0,0 +1,621 @@
+
+    .. tab-item:: Fields
+
+        Create the Fields by defining:
+
+        - The number of entities;
+        - The entities ids and location. Thus, the |Field| scoping
+
+            - If not specified, the location is *'nodal'* by default.
+            - Each entity (here, the nodes) must have a |Scoping| id. The ids allows DPF to apply an operator on the
+              corresponding entities. For more detailed explanation about the influence of the |Scoping| on the operations,
+              see the :ref:`ref_basic_maths_scoping_handling` section on this tutorial.
+
+        Import the necessary DPF modules.
+
+        .. jupyter-execute::
+
+            # Import the ``ansys.dpf.core`` module
+            from ansys.dpf import core as dpf
+            # Import the math operators module
+            from ansys.dpf.core.operators import math as maths
+
+        Create the Fields by intanciating the |Field| object.
+
+        .. jupyter-execute::
+
+            # Instantiate the Fields
+            num_entities = 2
+            field1 = field2 = field3 = field4 = dpf.Field(nentities=num_entities)
+
+            # Define the scoping ids
+            field1.scoping.ids = field2.scoping.ids = field3.scoping.ids = field4.scoping.ids = range(num_entities)
+
+            # Set the data to each Field
+            field1.data = [1.0, 2.0, 3.0, 4.0, 5.0, 6.0]
+            field2.data = [7.0, 3.0, 5.0, 8.0, 1.0, 2.0]
+            field3.data = [6.0, 5.0, 4.0, 3.0, 2.0, 1.0]
+            field4.data = [4.0, 1.0, 8.0, 5.0, 7.0, 9.0]
+
+            # Print the Fields
+            print("Field 1","\n", field1, "\n"); print("Field 2","\n", field2, "\n");
+            print("Field 3","\n", field3, "\n"); print("Field 4","\n", field4, "\n")
+
+    .. tab-item:: FieldsContainers
+
+        Create the FieldsContainers using the |fields_container_factory|.  Here, we use the |over_time_freq_fields_container|
+        function that creates a |FieldsContainer| with a *'time'* label.
+
+        .. jupyter-execute::
+
+            # Create the FieldsContainers
+            fc1 = dpf.fields_container_factory.over_time_freq_fields_container(fields=[field1, field2])
+            fc2 = dpf.fields_container_factory.over_time_freq_fields_container(fields=[field3, field4])
+
+            # Print the FieldsContainers
+            print("FieldsContainer1","\n", fc1, "\n")
+            print("FieldsContainer2","\n", fc2, "\n")
+
+
+To make the mathematics operations, we use the operators available in the |math operators| module.
+Their usage is similar, for each operation you must instantiate the operator and use ``.eval()`` method to compute
+and retrieve the results.
+
+Mathematical operations with Fields
+-----------------------------------
+
+.. tab-set::
+
+    .. tab-item:: Addition
+
+        Here, we use:
+
+        - The 'add' operator for component wise addition
+        - The 'accumulate' operator to find the total sum of each component for all the entities
```

Comment:
To highlight an inline piece of code, it is better to use backticks:

```suggestion
        - The `add` operator for component wise addition
        - The `accumulate` operator to find the total sum of each component for all the entities
```

Reply:
```suggestion
        - The |add| operator for component wise addition
        - The |accumulate| operator to find the total sum of each component for all the entities
```